library/general/src/lib/cfa/sysctl_config.rb

```diff
@@ -157,7 +157,9 @@ def file_paths_in(path)
       if Yast::FileUtils.IsFile(path)
         [path]
       elsif Yast::FileUtils.IsDirectory(path)
-        Yast::SCR.Read(Yast::Path.new(".target.dir"), path).map { |file| File.join(path, file) }
+        Yast::SCR.Read(Yast::Path.new(".target.dir"), path)
+          .select { |f| f.end_with?(".conf") }
```

Comment:
please write that note about this suffix also to this place, so we know in future why only .conf are selected.